File: Modul-3/Soal-5/PRAK305_2410817310012_FAISALTANJUNG.py
inputdetik = int(input())

hari = inputdetik // 86400 
jam = (inputdetik % 86400) // 3600
menit = (inputdetik % 3600) // 60
detik = inputdetik % 60

if hari > 0:
    print(f"{hari} hari {jam:02}:{menit:02}:{detik:02}")
else:
    print(f"{jam:02}:{menit:02}:{detik:02}")

File: Modul-3/Soal-5/PRAK305_2410817310012_FAISALTANJUNG.c
#include <stdio.h>
int main(){
    int inputdetik, hari, jam, menit, detik;
   
    printf("");
    scanf("%d", &inputdetik);

    hari = inputdetik / 86400 ;
    jam = (inputdetik % 86400) / 3600;
    menit = (inputdetik % 3600) / 60;
    detik = inputdetik % 60;

    if(hari > 0){
        printf("%.0d hari %02d:%02d:%02d\n",hari, jam , menit, detik);
    }
    else{
        printf("%02d:%02d:%02d\n",jam, menit, detik);
    }
}